prbt_hardware_support/test/unittests/unittest_speed_observer.cpp

```diff
@@ -148,14 +148,14 @@ MATCHER_P2(SpeedAtIGe, i, x,
            "Speed at index " + PrintToString(i) + std::string(negation ? "is not" : "is") + " greater or equal to" +
            PrintToString(x) + ".")
 {
-  return static_cast<unsigned long>(i) < arg->name.size() && arg->speed.at(i) >= x;
+  return static_cast<unsigned long>(i) < arg->name.size() && arg->speed.at(static_cast<unsigned long>(i)) >= x;
```

Comment:
```suggestion
  return static_cast<unsigned long>(i) < arg->name.size() && arg->speed.at(static_cast<size_t>(i)) >= x;
```
maybe?

Reply:
Here we can see why macros s***. Normally you would use the container datatype itself to request the correct type for the index. Here it is not possible because it is a macro.

Reply:
Done.